dbt-get-started/README.md

```diff
@@ -99,6 +137,30 @@ SHOW MATERIALIZED VIEWS;
 
 You'll notice that you're only able to `SELECT` from `avg_bid` — this is because it is the only materialized view! This view is incrementally updated as new data streams in, so you get fresh and correct results with low latency. Behind the scenes, Materialize is indexing the results of the embedded query in memory.
 
+### Continuous testing
+
+To validate that the schema storing the tests was created:
+
+```sql
+SHOW SCHEMAS;
+
+        name
+--------------------
+ public
+ public_etl_failure
+```
+
+, and that the materialized views that continuously test the `avg_bid` view for failures are up and running:
```

Comment:
The README is kind of minimal right now, but in a second iteration we could add an example that shows a test failing (this is probably easier if we base things off `cnt_ticker` rather than `avg_bid`?). Otherwise, since we don't control the data that is produced, this doesn't show much!